Build/AdditionalProjectTransforms.xslt

```diff
@@ -0,0 +1,5 @@
+<xsl:if test="/Input/Properties/DefineNET4DetectionConstant = 'True'">
+  <PropertyGroup>
+    <DefineConstants Condition=" '$(TargetFrameworkVersion)' == 'v4.0' ">NET4</DefineConstants>
```

Comment:
Yes, sounds better. Inverting this and detecting 4.5+ instead should be a better strategy.
I'll update the PR tomorrow.